exchanges/btse/btse_wrapper.go

```diff
@@ -712,12 +824,152 @@ func (b *BTSE) ValidateCredentials() error {
 	return b.CheckTransientError(err)
 }
 
+// FormatExchangeKlineInterval formats kline interval to exchange requested type
+func (b *BTSE) FormatExchangeKlineInterval(in kline.Interval) string {
+	return strconv.FormatFloat(in.Duration().Minutes(), 'f', 0, 64)
+}
+
 // GetHistoricCandles returns candles between a time period for a set time interval
 func (b *BTSE) GetHistoricCandles(pair currency.Pair, a asset.Item, start, end time.Time, interval kline.Interval) (kline.Item, error) {
-	return kline.Item{}, common.ErrFunctionNotSupported
+	if err := b.ValidateKline(pair, a, interval); err != nil {
+		return kline.Item{}, err
+	}
+
+	fPair, err := b.FormatExchangeCurrency(pair, a)
+	if err != nil {
+		return kline.Item{}, err
+	}
+	intervalInt, err := strconv.Atoi(b.FormatExchangeKlineInterval(interval))
+	if err != nil {
+		return kline.Item{}, err
+	}
+
+	klineRet := kline.Item{
+		Exchange: b.Name,
+		Pair:     fPair,
+		Asset:    a,
+		Interval: interval,
+	}
+
+	switch a {
+	case asset.Spot:
+		req, err := b.OHLCV(fPair.String(),
+			start,
+			end,
+			intervalInt)
+		if err != nil {
+			return kline.Item{}, err
+		}
+		for x := range req {
+			klineRet.Candles = append(klineRet.Candles, kline.Candle{
+				Time:   time.Unix(int64(req[x][0]), 0),
+				Open:   req[x][1],
+				High:   req[x][2],
+				Low:    req[x][3],
+				Close:  req[x][4],
+				Volume: req[x][5],
+			})
+		}
+	case asset.Futures:
+		return kline.Item{}, common.ErrNotYetImplemented
+	default:
+		return kline.Item{}, fmt.Errorf("asset %v not supported", a.String())
+	}
+
+	klineRet.SortCandlesByTimestamp(true)
```

Comment:
Despite this being here, its still descending. I also just noticed all other implementations use `ret.SortCandlesByTimestamp(false)`. Could you please change it to that? :D